### PR TITLE
Added SimpleCipher test cases and test class definion

### LINKED ISSUE #589: Add Simple Cipher exercise
Implement with solution Simple Cipher exercise

### PR DESCRIPTION
**Pull request by Bajger**
- fixes #589

### CODE REVIEW

**Comment by Bajger** [comment]
@glennj Please take a look at solution. This wasn't really my favorite exercise :) I believe there are nicer ways to implement  solution. T[ests are passing for this exercise](https://github.com/exercism/pharo-smalltalk/actions/runs/6201033872/job/16836902287?pr=590#step:4:1633). Will add rest of metadata (config and /exercise/simple-cipher later).

File: exercises/practice/simple-cipher/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [],
```
Comment (ErikSchierboom) [inline exercises/practice/simple-cipher/.meta/config.json:2]: You're missing an author

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/practice/simple-cipher/.meta/config.json (hunk 1 above)
Comment (Bajger) [reply exercises/practice/simple-cipher/.meta/config.json]: Thanks! Fixed.

**Review by glennj — APPROVED** [review]
Looks good. Of course the example solution does not need to be the "exemplar".

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**